lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -42,7 +42,7 @@ cloudauthz==0.2.0
 cloudbridge==2.0.0
 cmd2==0.8.9
 contextlib2==0.5.5 ; python_version < '3.5'
-cryptography==2.6.1
+cryptography==2.8
```

Comment:
I think you can also remove `asn1crypto` from this file, which is no more a dependency of `cryptography`: https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst#28---2019-10-16

Reply:
Sure.